data_structures/inorder.cpp

```diff
@@ -0,0 +1,96 @@
+/**
+ * @file
+ * @brief [Inorder Tree Traveral](https://www.javatpoint.com/inorder-traversal)
+ * @author [Chaitanya Giri](https://github.com/Chaitanya134)
+ */
+
+#include <iostream>  /// for IO operations
+#include <cassert>   /// for assert
+#include <queue>     /// for storing node in level order traversal
+using namespace std;
```

Comment:
This is discouraged. Please strictly use `std::` for each member instead. You can use `using std::cout;`, `using std::cin;`, and so on for all `std::` members.